### PR TITLE
Added android sample project

### PR DESCRIPTION
**Pull request by vcoolish**
## Description

Added Android sample project for integration guide in developers documentation

### CODE REVIEW

File: samples/android/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+See especially Wallet Core
+[Integration Guide](https://developer.trustwallet.com/wallet-core/integration-guide),
+and [Build Instructions](https://developer.trustwallet.com/wallet-core/building).
+
+## Building and Running
+
+Open in Android Studio and run. Result will appear in logs output and on device screen
+
+<img src="/art/screenshot" width="49%">
```
Comment (hewigovens) [inline samples/android/README.md]: use http url for image?

Comment (vcoolish) [reply samples/android/README.md]: thx, will add gist

File: samples/android/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ and [Build Instructions](https://developer.trustwallet.com/wallet-core/building)
 
 Open in Android Studio and run. Result will appear in logs output and on device screen
 
-<img src="/art/screenshot" width="49%">
+![Screenshot](https://gist.github.com/3ee8891d559e021683b58ffb37f72cf1)
```
Comment (hewigovens) [inline samples/android/README.md]: I think you can add that image to developer repo?

Comment (vcoolish) [reply samples/android/README.md]: you mean show in guide?

Comment (vcoolish) [reply samples/android/README.md]: I decided to remove it

Comment (hewigovens) [reply samples/android/README.md]: 👌

**Pull request merged**